api/src/pcapi/core/educational/api/stock.py

```diff
@@ -82,21 +83,41 @@ def edit_collective_stock(
     if booking_limit is None:
         check_booking_limit_datetime = stock.bookingLimitDatetime
 
-    offer_validation.check_booking_limit_datetime(stock, check_beginning, check_booking_limit_datetime)
-
-    # due to check_booking_limit_datetime the only reason beginning < booking_limit_dt is when they are on the same day
-    # in the venue timezone
-    if beginning is not None and beginning < updatable_fields["bookingLimitDatetime"]:
-        updatable_fields["bookingLimitDatetime"] = updatable_fields["beginningDatetime"]
-
     collective_stock_unique_booking = educational_models.CollectiveBooking.query.filter(
         educational_models.CollectiveBooking.collectiveStockId == stock.id,
         sa.sql.elements.not_(
             educational_models.CollectiveBooking.status == educational_models.CollectiveBookingStatus.CANCELLED
         ),
     ).one_or_none()
+
+    current_bookings = [
+        booking for booking in stock.collectiveBookings if booking.status != CollectiveBookingStatus.CANCELLED
+    ]
+    current_booking = current_bookings[0] if current_bookings else None
+    if "beginningDatetime" not in stock_data and "bookingLimitDatetime" not in stock_data:
+        price = updatable_fields.get("price")
+        if current_booking and current_booking.status == CollectiveBookingStatus.USED:
+            last_edition_date = current_booking.dateUsed + datetime.timedelta(days=2)  # type: ignore [operator]
+            if last_edition_date > datetime.datetime.utcnow() and price:  # type: ignore [operator]
+                validation.check_if_edition_lower_price_possible(stock, price)
+        elif current_booking and current_booking.status == CollectiveBookingStatus.CONFIRMED:
+            validation.check_collective_stock_is_editable(stock)
+            if price is not None:
+                validation.check_if_edition_lower_price_possible(stock, price)
+        elif current_booking and current_booking.status != CollectiveBookingStatus.USED or current_booking == None:
+            validation.check_collective_stock_is_editable(stock)
```

Comment:
tu peux simplifier avec: 
```suggestion
        else:
            validation.check_collective_stock_is_editable(stock)
```